tests/app/celery/test_tasks.py

```diff
@@ -519,39 +499,13 @@ def test_process_smss_job_metric(self, sample_template_with_placeholders, mocker
         assert persisted_notification[0].personalisation == {"name": "Jo"}
         assert persisted_notification[0]._personalisation == signer.sign({"name": "Jo"})
         assert persisted_notification[0].notification_type == SMS_TYPE
-        assert pbsbp_mock.assert_called_with(mock.ANY, 1) is None
+        assert pbsbp_mock.assert_called_with(mock.ANY, 1, notification_type="sms", priority="normal") is None
 
 
 class TestProcessJob:
     def test_should_process_sms_job_FF_PRIORITY_LANES_true(self, sample_job, mocker):
-        mocker.patch.object(Config, "FF_PRIORITY_LANES", True)
-        mocker.patch("app.celery.tasks.s3.get_job_from_s3", return_value=load_example_csv("sms"))
-        mocker.patch("app.celery.tasks.save_sms.apply_async")
-        mocker.patch("app.encryption.CryptoSigner.sign", return_value="something_encrypted")
-        mocker.patch("app.celery.tasks.create_uuid", return_value="uuid")
-
-        redis_mock = mocker.patch("app.celery.tasks.statsd_client.timing_with_dates")
-
-        process_job(sample_job.id)
-        s3.get_job_from_s3.assert_called_once_with(str(sample_job.service.id), str(sample_job.id))
-        assert signer.sign.call_args[0][0]["to"] == "+441234123123"
-        assert signer.sign.call_args[0][0]["template"] == str(sample_job.template.id)
-        assert signer.sign.call_args[0][0]["template_version"] == sample_job.template.version
-        assert signer.sign.call_args[0][0]["personalisation"] == {"phonenumber": "+441234123123"}
-        assert signer.sign.call_args[0][0]["row_number"] == 0
-        tasks.save_sms.apply_async.assert_called_once_with(
-            (str(sample_job.service_id), "uuid", "something_encrypted"), {}, queue=QueueNames.NORMAL_DATABASE
-        )
-        job = jobs_dao.dao_get_job_by_id(sample_job.id)
-        assert job.job_status == "finished"
-        assert job.processing_started is not None
-        assert job.created_at is not None
-        redis_mock.assert_called_once_with("job.processing-start-delay", job.processing_started, job.created_at)
-
-    def test_should_process_sms_job_FF_PRIORITY_LANES_false(self, sample_job, mocker):
-        mocker.patch.object(Config, "FF_PRIORITY_LANES", False)
         mocker.patch("app.celery.tasks.s3.get_job_from_s3", return_value=load_example_csv("sms"))
-        mocker.patch("app.celery.tasks.save_sms.apply_async")
+        mocker.patch("app.celery.tasks.save_smss.apply_async")
```

Comment:
Interesting! so before we were testing something like Priority Lanes on but batch saving off? That's messed up.